src/containers/BuildUnsignedSweepCoin/HederaForm.tsx

```diff
@@ -0,0 +1,141 @@
+import { Form, FormikHelpers, FormikProvider, useFormik } from 'formik';
+import { Link } from 'react-router-dom';
+import * as Yup from 'yup';
+import { Button, FormikTextfield } from '~/components';
+
+const validationSchema = Yup.object({
+  backupKey: Yup.string().required(),
+  backupKeyId: Yup.string(),
+  userKey: Yup.string().required(),
+  userKeyId: Yup.string(),
+  recoveryDestination: Yup.string().required(),
+  rootAddress: Yup.string().required(),
+  maxFee: Yup.number().optional(),
+  startTime: Yup.number()
+    .required()
+    .min(Date.now() / 1000, 'Start time must be in the future'),
+  nodeId: Yup.string().optional(),
+}).required();
+
+export type HederaFormProps = {
+  onSubmit: (
+    values: HederaFormValues,
+    formikHelpers: FormikHelpers<HederaFormValues>
+  ) => void | Promise<void>;
+};
+
+type HederaFormValues = Yup.Asserts<typeof validationSchema>;
+
+export function HederaForm({ onSubmit }: HederaFormProps) {
+  const formik = useFormik<HederaFormValues>({
+    onSubmit,
+    initialValues: {
+      userKey: '',
+      userKeyId: '',
+      backupKey: '',
+      backupKeyId: '',
+      recoveryDestination: '',
+      rootAddress: '',
+      maxFee: undefined,
+      nodeId: undefined,
```

Comment:
nit: optional fields should init to `null`

Reply:
expected values from the validation schema are are number or undefined